src/components/Input/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import "./Input.css";
+
+interface InputProps {
+  id: number;
+  placeholder: string;
+  onChange: (value: string) => void;
+}
+
+function Input({ id, placeholder, onChange }: InputProps) {
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Great job on figuring this part out. You could simplify this by mimicking the way that it was done on the basic `<input>` before. Like this: `onChange={(e) => setTodoText(e.currentTarget.value)}`, except you would replace the setTodoText with your prop onChange.